cmake/OmimTesting.cmake

```diff
@@ -1,5 +1,8 @@
 include(OmimConfig)
 
+# Enables ctest -T memcheck with valgrind
+include(CTest)
```

Comment:
Ok, so should we call -T memcheck in yaml flows?

Reply:
It can be a separate PR, @AndrewShkrob can you help here?

Reply:
It's not a suitable location. Please move it to root/CMakeLists.txt:
```cmake
...
if (NOT SKIP_TESTS)
  include(CTest)
  set(SKIP_TESTS FALSE)
  enable_testing()
  set(CTEST_CONFIGURATION_TYPE ${CMAKE_BUILD_TYPE})
endif()
```

Reply:
> It can be a separate PR

Why not make it now? What's the reason for these changes if they are useless? Almost nobody runs our tests locally, and absolutely nobody will run them with this flag.

Reply:
Because it requires time that I don't have now.